### ISSUE TITLE
how long  to train a model?

### ISSUE THREAD

**Issue by Aurora11111** [meijieru/crnn.pytorch#156]

**Comment by SreenijaK** [comment]
@Aurora11111 can you help me here?
I want to train with my own dataset. First I used the create_dataset.py and it created two files- data.mdb and lock.mdb.
Now i gave the same path of data.mdb to train and validate. If this is not right How do i split to train and validate.

And when i run train.py, I get the following error

File "train.py", line 60, in 
train_dataset = dataset.lmdbDataset(root=opt.trainRoot)
File "/home/ramu_yarru/ctpn/crnn.pytorch/dataset.py", line 25, in init
meminit=False)
lmdb.Error: /home/ramu_yarru/ctpn/crnn_train/data/train/data.mdb: Not a directory

What exactly is the input to train.py ?

**Comment by tabsun** [comment]
[0/60][4233/78125] Loss: 4.147334
[0/60][4234/78125] Loss: 6.068669
[0/60][4235/78125] Loss: 3.259431
[0/60][4236/78125] Loss: 5.895171
[0/60][4237/78125] Loss: 8.373442
[0/60][4238/78125] Loss: 3.521796
[0/60][4239/78125] Loss: 7.036237
[0/60][4240/78125] Loss: 6.868206
[0/60][4241/78125] Loss: 9.916059
........
Basically each step will take 6~8 seconds, is this normal?
